problem1.py

```diff
@@ -1 +1,20 @@
 
+#Enter Wins
+print("Enter your wins and losses for your team:")
+
+for i in range(6):
+    results = input("")
+
+
+
+if wins >= 5 :
```

Comment:
you're checking for wins, but `wins` is never updated or assigned.  an error occurs here.